nvtabular/ops/categorify.py

```diff
@@ -277,7 +285,17 @@ def apply_op(
                 new_gdf[name] = new_gdf[new_col]
                 new_gdf.drop(columns=[new_col], inplace=True)
 
+        if self.num_buckets is not None and self.freq_threshold == 0:
+            hash_bucket_op = HashBucket(self.num_buckets)
+            new_gdf = hash_bucket_op.apply_op(new_gdf, columns_ctx, "categorical")
```

Comment:
Whats the use case for doing the full categorical encoding and then hashing the output?  (like  do we need to incur the expense of doign the categorical encoding first if we are just going to hash everything after?)

Reply:
@benfred good point. you are right. I am working on that part. The example for that would be the case where `freq_threshold >1 + num_buckets >1`.